From baaed1ec3c6870ee7a15659543006cb07796cdf9 Mon Sep 17 00:00:00 2001
From: Dan Kephart <dan.kephart@lairdtech.com>
Date: Thu, 18 Feb 2016 12:15:27 -0500
Subject: [PATCH] laird: ath6kl: Change btcoex params from int to char to fix
 warning

For btcoex_chip_type and btcoex_ant_type, both are specified as byte for
the module parameter but unsigned int as the variable.  Fix variable to be
unsigned char to match parameter type of byte.

Bug: 9053
---
 drivers/net/wireless/ath/ath6kl/core.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/wireless/ath/ath6kl/core.c b/drivers/net/wireless/ath/ath6kl/core.c
index 9914fb5cf758..5371907022ff 100644
--- a/drivers/net/wireless/ath/ath6kl/core.c
+++ b/drivers/net/wireless/ath/ath6kl/core.c
@@ -39,8 +39,8 @@ static unsigned int ath6kl_p2p;
 static unsigned int testmode;
 static unsigned int recovery_enable;
 static unsigned int heart_beat_poll;
-static unsigned int btcoex_chip_type;
-static unsigned int btcoex_ant_config;
+static unsigned char btcoex_chip_type;
+static unsigned char btcoex_ant_config;
 
 module_param(debug_mask, uint, 0644);
 module_param(suspend_mode, uint, 0644);
-- 
2.11.0

